bccsp/pkcs11/pkcs11.go

```diff
@@ -687,8 +704,14 @@ func (csp *Provider) clearCaches() {
 }
 
 func (csp *Provider) findKeyPairFromSKI(session pkcs11.SessionHandle, ski []byte, keyType keyType) (pkcs11.ObjectHandle, error) {
+	keyID := ski
```

Comment:
And what about the layers above this? This is changing the ski stored in the cache but doesn't do anything about how the key is managed in `GetKey` or `getECKey`.